test/e2e/chaos_test.go

```diff
@@ -70,12 +71,17 @@ func testKillRandomMembers(parentCtx context.Context, client crclient.Client, cl
 			},
 			Data: map[string]string{"value": string(value)},
 		}
-		err := wait.PollImmediateUntil(5*time.Second, func() (bool, error) {
+		var previousCreationError string
+		err := wait.PollUntilContextTimeout(ctx, 5*time.Second, 30*time.Minute, true, func(ctx context.Context) (bool, error) {
 			if err := guestClient.Create(ctx, cm); err != nil {
+				if err.Error() != previousCreationError {
+					t.Logf("failed to create marker ConfigMap: %v", err)
```

Comment:
It might be good to log something like `failed to create marker ConfigMap: %v, continuing to poll` to indicate you haven't failed the testcase yet